src/VoyagerServiceProvider.php

```diff
@@ -117,7 +117,11 @@ protected function registerViewComposers()
      */
     protected function addStorageSymlinkAlert()
     {
-        $currentRouteAction = app('router')->current()->getAction();
+        if (app('router')->current() !== null) {
+            $currentRouteAction = app('router')->current()->getAction();
```

Comment:
But since your working on a service provider, you have access to `$this->app` hence `$this->app->router` is already instantiated.